packages/react-devtools-shared/src/backend/agent.js

```diff
@@ -635,28 +635,24 @@ export default class Agent extends EventEmitter<{|
     appendComponentStack,
     breakOnConsoleErrors,
     showInlineWarningsAndErrors,
+    hideDoubleLogsInStrictMode,
+    __REACT_DEVTOOLS_SHOW_INLINE_WARNINGS_AND_ERRORS__,
   }: {|
     appendComponentStack: boolean,
     breakOnConsoleErrors: boolean,
     showInlineWarningsAndErrors: boolean,
+    hideDoubleLogsInStrictMode: boolean,
   |}) => {
     // If the frontend preference has change,
     // or in the case of React Native- if the backend is just finding out the preference-
-    // then install or uninstall the console overrides.
+    // then reinstall the console overrides.
     // It's safe to call these methods multiple times, so we don't need to worry about that.
-    if (
-      appendComponentStack ||
-      breakOnConsoleErrors ||
-      showInlineWarningsAndErrors
-    ) {
-      patchConsole({
-        appendComponentStack,
-        breakOnConsoleErrors,
-        showInlineWarningsAndErrors,
-      });
-    } else {
-      unpatchConsole();
-    }
```

Comment:
~~Why did we remove this, by the way? I think it's still valid?~~

~~The case would be when we eagerly patch on the React Native side (because of default preference values) but then once we load actual user preferences (which we do async) we realize that we don't need to patch b'c they're all false- so then we unpatch.~~

Nevermind. I forgot that we're _always_ overriding now and either hiding or dimming logs. 😄